backend/xivdpscalc/pps/sch.py

```diff
@@ -2,7 +2,9 @@
 
 import math
 
+from _collections import defaultdict
```

Comment:
Out of curiosity, why import from the C implementation?

Reply:
o lol thats what i get for trusting eclipse to auto import things